docs/src/main/sphinx/optimizer/pushdown.rst

```diff
@@ -151,6 +151,52 @@ to perform the remaining query processing on a smaller amount of data.
 The specifics for the supported pushdown of table joins varies for each data
 source, and therefore for each connector.
 
+However, there are some generic conditions that must be met in order for a join
+to be pushed down:
+
+* join pushdown must be enabled in a catalog
```

Comment:
This one is a bit strange, since this is a requirement for specific connectors that have a configuration flag to enable join pushdown. Also, I think the choice to disable by default for JDBC connectors is wrong -- I suspect that most joins do not increase the cardinality.

We could either

* remove this point and add a sentence below that says something like "A connector may require additional configuration to enable join pushdown."
* make it enabled by default for JDBC connectors and then there is no motivation to discuss this here at all